core/watcher/eventsource/value.go

```diff
@@ -20,80 +20,115 @@ type ValueWatcher struct {
 	*BaseWatcher
 
 	out        chan struct{}
-	namespace  string
-	changeMask changestream.ChangeType
-	filter     func(changestream.ChangeEvent) bool
+	filterOpts []changestream.SubscriptionOption
```

Comment:
This `ValueWatcher` declaration is a long way from its first reference down below.

I also think it is now a poor name. The original intent was that it was a watcher for one value/row, but it can be for anything now based on the filters. How about just `NotifyWatcher`?

Reply:
Yes.